core/spec/lib/search/base_spec.rb

```diff
@@ -36,6 +36,26 @@
     end
   end
 
+  context "when ascend_by_master_price scope is included in the initalization params" do
+    let(:params) { { search: { ascend_by_master_price: nil } } }
+
+    subject { described_class.new(params).retrieve_products }
+
+    it "returns product in ascending order" do
+      expect(subject.map { |product| product.price.to_i }).to eq [9, 11, 16]
+    end
+  end
+
+  context "when descend_by_master_price scope is included in the initalization params" do
+    let(:params) { { search: { descend_by_master_price: nil } } }
+
+    subject { described_class.new(params).retrieve_products }
+
+    it "returns products witout errors" do
```

Comment:
I think this should say "returns products in descending order" instead, since that's the format used in the other.